pkg/asset/templates/kube-apiserver.yaml

```diff
@@ -5,13 +5,11 @@ metadata:
   namespace: kube-system
   labels:
     k8s-app: kube-apiserver
-    version: v1.3.0-beta.2
```

Comment:
I don't see how this is an issue especially given you're deleting the entire daemonset anyway -- there shoulldn't be anything to conflict with